### ISSUE TITLE
Not loading multiple themes

### ISSUE THREAD

**Issue by laujonat** [echoulen/storybook-addon-styled-component-theme#25]
Hi, I'm having some difficulty loading the themes into my storybook.  
Going through the documentation, I was able to set up the package with the add on correctly instantiated. 

<img width="789" alt="Screen Shot 2019-05-17 at 11 49 53 PM" src="https://user-images.githubusercontent.com/3301828/57965844-96e30f00-78fe-11e9-919b-74175b4da9ad.png">

config.js
```
import { configure, addDecorator } from "@storybook/react";
import { withInfo } from "@storybook/addon-info";
import { withThemesProvider } from 'storybook-addon-styled-component-theme';

import GlobalStyle from "./decorators/globalStyle";
import themes from "../src/components/common/themes";

const demoThemes = [
  themes.default,
  themes.dark
]

addDecorator(
  withInfo({
    inline: true,
    header: false,
    source: false,
    maxPropsIntoLine: 1
  })
);
addDecorator(GlobalStyle);
addDecorator(withThemesProvider(demoThemes))
const req = require.context('../src', true, /\.stories\.js$/); // <- import all the stories at once
function loadStories() {
  req.keys().forEach(filename => req(filename));
}

configure(loadStories, module)
```
addons.js
```
import '@storybook/addon-actions/register'
// import '@storybook/addon-viewport/register';
// import '@storybook/addon-knobs/register'
import "@storybook/addon-storysource/register"
import '@storybook/addon-links/register'
import 'storybook-addon-styled-component-theme/dist/src/register'; // v1.1.0^
```

`storybook-addon-styled-component-theme": "^1.2.0"`
On the main Storybook, I am able to see the Themes map directly to the number of elements in the array of options.  

Once I trigger a theme change, the above error occurs. 
I know the theme is loaded into the storyboard because if I put a theme that is invalid, the following error occurs.

<img width="789" alt="Screen Shot 2019-05-17 at 11 49 53 PM" src="https://user-images.githubusercontent.com/3301828/57965894-6b145900-78ff-11e9-9c05-74b6f1099660.png">

Is this a reload issue?
Any advice would be appreciated.

**Comment by echoulen** [comment]
make sure your storybook is `v5` and
please install the latest version `v1.2.3`

**Comment by laujonat** [comment]
Thank you for the response. 
Here is the list of dependencies from my package.json. 
```
    "@babel/cli": "^7.4.3",
    "@babel/core": "^7.4.3",
    "@babel/plugin-proposal-class-properties": "^7.4.4",
    "@babel/preset-env": "^7.4.3",
    "@babel/preset-react": "^7.0.0",
    "@storybook/addon-actions": "^5.0.11",
    "@storybook/addon-info": "^5.0.11",
    "@storybook/addon-links": "^5.0.11",
    "@storybook/addon-options": "^5.0.11",
    "@storybook/addon-storysource": "^5.0.11",
    "@storybook/react": "^5.0.11",
    "babel-loader": "^8.0.5",
    "babel-plugin-styled-components": "^1.10.0",
    "cross-env": "^5.2.0",
    "eslint-config-prettier": "^4.3.0",
    "eslint-plugin-html": "^5.0.5",
    "eslint-plugin-import": "^2.17.2",
    "eslint-plugin-prettier": "^3.1.0",
    "eslint-plugin-react": "^7.13.0",
    "lint-staged": "^8.1.6",
    "npm": "^6.9.0",
    "prettier": "^1.17.1",
    "prop-types": "^15.7.2",
    "react": "^16.8.2",
    "react-dom": "^16.8.2",
    "react-scripts": "^3.0.1",
    "react-test-renderer": "^16.8.6",
    "rollup": "^1.12.2",
    "rollup-plugin-babel": "^4.3.2",
    "rollup-plugin-commonjs": "^10.0.0",
    "rollup-plugin-node-resolve": "^5.0.0",
    "rollup-plugin-replace": "^2.1.1",
    "storybook-addon-styled-component-theme": "^1.2.3",
    "styled-components": "^4.1.3"
```

Here is the list of peer dependencies.
```
 "react": "^16.8.6",
"react-dom": "^16.8.6",
"styled-components": "^2.0.0 || ^3.0.2"
```

Still experiencing the same issue. In the array of themes, only the first item of the list will be rendered onto the storybook, but I can investigate a bit further.

**Comment by echoulen** [comment]
@laujonat 
may you show the themes object?

**Comment by laujonat** [comment]
Here's my theme.js.
```
const themes = {}

themes.default = {
  text: "#333333",
  success: "#0F9960",
  alert: "C23030",
  warn: "#F2B824",
}

themes.stealth = {
  text: "#C23030",
  success: "#C23030",
  alert: "#C23030",
  warn: "#C23030",
}

export default themes
```

And config.js

```
import { configure, addDecorator } from "@storybook/react"
import { withInfo } from "@storybook/addon-info"
import { withThemesProvider } from "storybook-addon-styled-component-theme"

import themes from "../src/shared/themes"

import GlobalStyle from "./decorators/globalStyle"

const _themes = [themes.default, themes.stealth]

addDecorator(GlobalStyle)
addDecorator(withThemesProvider(_themes)
```

Thank you

**Comment by echoulen** [comment]
https://github.com/echoulen/storybook-addon-styled-component-theme#remind

**Comment by laujonat** [comment]
@echoulen I'm sorry, unclear what you are referring to. The project is set up according to the specifications.

Here is the repository: https://github.com/laujonat/styled-lite

**Comment by echoulen** [comment]
modify your themes to

```
themes.default = {
  name: "default",
  text: "#333333",
  success: "#0F9960",
  alert: "C23030",
  warn: "#F2B824",
}

themes.stealth = {
  name: "stealth",
  text: "#C23030",
  success: "#C23030",
  alert: "#C23030",
  warn: "#C23030",
}
```

**Comment by laujonat** [comment]
@echoulen thank you, it works this great!  May I suggest rewording`Make sure every theme with name property` to maybe `Include a name attribute in the theme object`?   

Anyways, I understand now. Thank you.
